examples/nlp/machine_translation/noisy_channel_reranking.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Given NMT model's .nemo file, this script can be used to translate text.
+USAGE Example:
+1. Obtain text file in src language. You can use sacrebleu to obtain standard test sets like so:
+    sacrebleu -t wmt14 -l de-en --echo src > wmt14-de-en.src
+2. Translate:
+    python nmt_transformer_infer.py --model=[Path to .nemo file] --srctext=wmt14-de-en.src --tgtout=wmt14-de-en.pre
+"""
```

Comment:
can we also add a link to the NCR paper in the docstring

Reply:
This has been added, thanks!